src/main/java/edu/harvard/iq/dataverse/api/Datasets.java

```diff
@@ -2691,9 +2689,12 @@ public Response listLocks(@QueryParam("type") String lockType, @QueryParam("user
             try {
                 lockTypeValue = DatasetLock.Reason.valueOf(lockType);
             } catch (IllegalArgumentException iax) {
-                String validValues = Strings.join(",", DatasetLock.Reason.values());
+                StringJoiner reasonJoiner = new StringJoiner(", ");
+                for (Reason r: Reason.values()) {
+                    reasonJoiner.add(r.name());
+                };
                 String errorMessage = "Invalid lock type value: " + lockType + 
-                        "; valid lock types: " + validValues;
+                        "; valid lock types: " + reasonJoiner.toString();
```

Comment:
This refactoring seems completely unrelated to poi/tika updates. @qqmyers can you please either revert this or explain under "how to test" how to exercise this code? Thanks.

Reply:
FWIW: import com.beust.jcommander.Strings; was only available through the older versions of poi/tika

Reply:
Ah, makes more sense now. Thanks for explaining and for the update to "how to test".